### ISSUE TITLE
[Bug] Docker Integration doesn't work with Podman

### ISSUE THREAD

**Issue by leruaa** [benphelps/homepage#254]
I've configured the docker integration for my services and I see the little green dot, but when I click on it, I have the following error:

> Application error: a client-side exception has occurred (see the browser console for more information).

Looking at the browser console I see:

```
TypeError: Cannot read properties of undefined (reading 'rx_bytes')
    at D (index-f387f442492a0fa8.js:1:8855)
    at Eo (framework-b3ca2e9a9574e304.js:1:63191)
    at Pu (framework-b3ca2e9a9574e304.js:1:74879)
    at xi (framework-b3ca2e9a9574e304.js:1:121314)
    at bs (framework-b3ca2e9a9574e304.js:1:109367)
    at gs (framework-b3ca2e9a9574e304.js:1:109295)
    at vs (framework-b3ca2e9a9574e304.js:1:109158)
    at as (framework-b3ca2e9a9574e304.js:1:105938)
    at ls (framework-b3ca2e9a9574e304.js:1:104489)
    at S (framework-b3ca2e9a9574e304.js:1:138448)
```

I think the error is from there: https://github.com/benphelps/homepage/blob/0b43f83daabae5655c1c950377e37fd1ca3f0631/src/components/services/widgets/service/docker.jsx#L57

When I call the following API (corresponding to one of my services : `/api/docker/stats/gitea/my-docker`, I get that:

```json
      "networks":{
         "network":{
            "rx_bytes":8465590,
            "rx_packets":0,
            "rx_errors":0,
            "rx_dropped":0,
            "tx_bytes":41127436,
            "tx_packets":0,
            "tx_errors":0,
            "tx_dropped":0
         }
      }
```

Looks like the network is expected to be `eth0`, but with Podman, it's `network`.

It would be great if we can support Podman :)

**Comment by benphelps** [comment]
Thank you for the details, we'll get this fixed in the next release.

**Comment by benphelps** [comment]
This will be fixed once fe7fa5c goes out.
